android/src/main/java/com/swmansion/rnscreens/ext/ViewExt.kt

```diff
@@ -40,6 +40,9 @@ internal fun View.isInsideScrollViewWithRemoveClippedSubviews(): Boolean {
     }
     var parentView = this.parent
     while (parentView is ViewGroup && parentView !is ScreenStack) {
+        if (parentView is ReactHorizontalScrollView) {
```

Comment:
Maybe add it as `||` in the second if?

Reply:
I tried it, but got a compilation error because of undeclared `removeClippedSubviews`. It seems like accessing specific properties is not possible when looking for multiple types with `is` in Kotlin

Reply:
I see great usecase for `when` cause then

Reply:
https://github.com/software-mansion/react-native-screens/pull/2403/commits/1ccaa42fc7afc86496dfdd93347408246c145d17